game/player.py

```diff
@@ -18,8 +18,11 @@ def __init__(self, user: discord.Member):
 
     # generates the role's PM
     def role_pm(self):
-        return (f'Hello {self.user}, you are a **{self.display_role}**. {self.role.description}'
-                f'\nWin Condition: {self.faction.win_con}')
+        return (
+            f'Hello {self.user}, you are a **{self.display_role}**.'
+            f'{self.role.description}'  # This follows the previous line.
```

Comment:
there should be a space between these lines

Reply:
I fixed it.